tools/block-generator/Makefile

```diff
@@ -0,0 +1,32 @@
+SCENARIO = scenarios/config.app.create.yml # test_config.yml
+SKIP = --skip-runner
+RESETDB = # --reset-db
+REPORTS = RUN_RUNNER_OUTPUTS
+
+debug-blockgen:
+	python run_runner.py \
+	--conduit-binary ./conduit \
+	--scenario $(SCENARIO) \
+	--report-directory $(REPORTS) \
```

Comment:
why not write report to go-algorand/tmp/? it's included in .gitignore.

Reply:
7534b577ccdde66fdc395df03db5cbe3b0625f91